src/main/java/com/google/codeu/data/Location.java

```diff
@@ -29,9 +42,30 @@ public String getDescription() {
         return description;
     }
 
-    public UUID getID() {
+    public String getID() {
         return id;
     }
 
+    public String getImageURL() {
+        return imageUrl;
+    }
+
+    public String getImageLabels() {
+        return imageLabels;
+    }
+
+
```

Comment:
Again, not sure if the extra lines are ok. Could you double check? @jonathantanmy

Reply:
This is fine, since you're changing adjacent lines (as I described above).